ocaml/xapi/xapi_database_backup.ml

```diff
@@ -0,0 +1,175 @@
+module D=Debug.Make(struct let name="xapi_database_backup" end)
+open D
+
+open Http
+open Xmlrpc_client
+open Db_cache_types
+open Stdext
+open Threadext
+
+type stat = {created : int64; modified : int64; deleted : int64;} [@@deriving rpc]
```

Comment:
This is identical to `Db_cache_types.Stat.t`, correct? But with rpc-deriving added to it.

Reply:
Yes. I can't seem to get it to work with `Db_cache_types.Stat.t`, I can't make that type work with deriving rpc